checker/localizations_src/en/messages.yaml

```diff
@@ -72,10 +72,19 @@ request-property-became-not-nullable: the request property %s became not nullabl
 request-body-became-not-nullable: the request's body became not nullable
 request-property-became-enum: request property %s was restricted to a list of enum values
 request-property-enum-value-removed: removed the enum value %s of the request property %s
+request-property-enum-value-added: added the new '%s' enum value to the request property %s
 request-body-max-decreased: the request's body max was decreased to %s
 request-property-max-decreased: the %s request property's max was decreased to %s
 request-body-max-length-decreased: the request's body maxLength was decreased to %s
 request-property-max-length-decreased: the %s request property's maxLength was decreased to %s
+request-body-min-length-increased: the request's body maxLength was increased to %s
+request-property-min-length-increased: the %s request property's maxLength was increased to %s
+request-body-min-length-decreased: the request's body maxLength was decreased to %s
+request-property-min-length-decreased: the %s request property's maxLength was decreased to %s
+request-body-max-increased: the request's body max was increased to %s
+request-property-max-increased: the %s request property's max was increased to %s
+request-body-max-length-increased: the request's body maxLength was increased to %s
+request-property-max-length-increased: the %s request property's maxLength was increased to %s
 request-body-max-length-set: the request's body maxLength was set to %s
```

Comment:
I suggest to add the original value to the message, for example:

the %s request property's maxLength was increased from %s to %s